### ISSUE TITLE
Missing words in string? Clear private data.

### ISSUE THREAD

**Issue by havarh** [mozilla/firefox-ios#417]
When translating Firefox for iOS i found an english string that I think are missing a couple of words.
In the file: **Client/Frontend/Settings/SettingsTableViewController.swift** line **384**
The line reads:
```Swift
var message: String { return NSLocalizedString("Are you sure you want all of your data? This will also close all open tabs.", tableName: "ClearPrivateData", comment: "Message shown in the dialog when clearing everything")
```
Shouldn't it read "Are you sure you want **to clear** all of your data? […]"?

**Comment by ncalexan** [comment]
Good catch!  @wesj, @flod: can you address this?

**Comment by wesj** [comment]
We fixed this in PR #413. Also, I need better commit messages there... Next time.
